ocm_python_wrapper/cluster.py

```diff
@@ -256,33 +257,85 @@ def addon_info(self):
             self.addon_name
         ).to_dict()
 
-    def validate_addon_parameters(self, parameters):
+    def filter_required_addon_parameters(self, parameters=None):
```

Comment:
`filter_required_addon_parameters` - if this filters required parameters then "required" should be done here

Reply:
Done